src/browser/provider/built-in/base.js

```diff
@@ -41,7 +41,7 @@ export default class BrowserProviderBase {
         this.resizeCorrections[id] = { width: newWidth - width, height: newHeight - height };
     }
 
-    async resizeWindow (id, pageInfo, width, height) {
+    async resizeWindow (id, width, height, pageSize) {
```

Comment:
Then I don't understand why we should have `width`, `height`, `currentPageWidth`, `currentPageHeight` instead of  `width`, `height`, `currentWidth`, `currentHeight`